### ISSUE TITLE
Shops are not recognized as POI 

### ISSUE THREAD

**Issue by tommycrock** [osmandapp/OsmAnd#20506]
### Description

Shops with a tag that osmand doesn't recognise don't display the opening hours on the POI popup card. e.g. https://www.openstreetmap.org/way/1294840099 has shop=catalogue, which is shown as a shop but the opening hours are missing.

### Steps to reproduce

Navigate to https://www.openstreetmap.org/way/1294840099 (Yorkshire, England) and click it.

### Actual result

The card is shown but there's no opening hours displayed.

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/937322de-f7ed-4dc0-9c6d-5110af23f856"  width="300" height="auto"/> </th>
  </tr>
</table>

### Expected result

Opening hours displayed 

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: 4.8.4

**Comment by scaidermern** [comment]
Same bug for this shop: https://www.openstreetmap.org/node/6812096181

**Comment by vshcherb** [comment]
Reproduced, if you click edit POI - information is parsed correctly however it's not displayed whether POI open or not

**Comment by scaidermern** [comment]
In my case no information is displayed at all (no name, no website, no opening hours). Except for the URL to the OSM node.

**Comment by tommycrock** [comment]
Yes, I also see no name (just location). My example doesn't have other tags but presumably also missing

**Comment by vshcherb** [comment]
We need to index all shops as POI may be make a generic-catch category somehow and encode specific type as text

**Comment by vshcherb** [comment]
To do: on generation site we need to add "catch-up" tag in poi_types.xml, so if shop is not one of listed, it converts to ```shop=general``` + ```shop:type=catalogue```

**Comment by dmpr0** [comment]
## UI Review

### iOS

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th>iOS</th>
        <th>Android</th>
      </tr>
      <tr>
        <td><video src="https://github.com/user-attachments/assets/639ea7a0-ac2c-4838-a6da-8253912862c1"  width="300" height="auto"/> </td>
        <td><video src="https://github.com/user-attachments/assets/f3074070-60ff-419c-a893-5fac14ee27a1"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   


- [x] **Search more** button doesn't work in **Nearby POI "Shop"**

**Comment by Zirochkabila** [comment]
# Testing 1

> OsmAnd Maps 5321 (Released: 2026-01-19)

### **Severity**: 🟠 Major / **Priority**: 🟠 Normal

* [ ] 1. Nearby POI “Shop” is not displayed in the point context menu

<details><summary>Details</summary>
<img width="900" alt="Screenshot" src="https://github.com/user-attachments/assets/d649bbed-3bbb-4acf-b077-9a824012a56d">
</details>

**Comment by vshcherb** [comment]
Nearby POI not supported as custom search as well, you would need to search all shops
